### PR TITLE
do not display command for default mappings (fixes #25)

### LINKED ISSUE #25: How do I stop displaying command names?
Is there a way to turn off the display of the command name?

image

(:VimTodoListsGoToPreviousItem in the picture)

### PR DESCRIPTION
**Pull request by jeyj0**
Adding `<silent>` to the mappings makes the commands not show up under the status bar. This fixes Issue #25.

### CODE REVIEW

File: plugin/vim-todo-lists.vim [hunk 1]
```diff
@@ -417,18 +431,18 @@ endfunction
...
 function! VimTodoListsDecreaseIndent()
-  normal! <<
+  normal! <<6l
 endfunction
 
 " Migrates file to new format
 function! VimTodoListsMigrate()
   normal! mz
-  silent! execute ':%s/^\(\s*\)\(\[.\]\)/\1- \2/'
+  silent! execute ':%s/^\(\s*\)\(\[.\]\)/\1\* \2/'
```
Comment (theg4sh) [inline plugin/vim-todo-lists.vim]: After switch to md-like syntax, syntax with `- [ ]` will be also deprecated, so:
```suggestion
  silent! execute ':%s/^\(\s*\)\(- \|\)\(\[.\]\)/\1\* \3/'
```

**Comment by breuerfelix** [comment]
i would love to get this merged !

**Comment by aserebryakov** [comment]
@jeyj0, I more I think about it, the whole idea of backward compatibility with previous versions seem to be essentially wrong for such essentially simple plugin.
So, I think the best solution is to have as many things as it possible configurable with some defaults that are aligned with markdown.

WDYT?

**Comment by jeyj0** [comment]
Makes sense, but since I converted to doom emacs, I am not going to implement it ;) Sounds cool though! :D

**Comment by aserebryakov** [comment]
> Makes sense, but since I converted to doom emacs, 

How could you?! =)

**Comment by breuerfelix** [comment]
@jeyj0 well but its not 'implementing' it... you just have to press the merge button :D

btw @aserebryakov i got a fork of this repo with this PR merged and some other modifications :) so you can instead just load this plugin if you want to :)

**Comment by aserebryakov** [comment]
de-facto merged into dev branch.

**Pull request closed without merging**